pkg/agent/proxy/proxier.go

```diff
@@ -276,6 +281,9 @@ func (p *proxier) installServices() {
 		}
 
 		p.serviceInstalledMap[svcPortName] = svcPort
+		for _, endpoint := range endpointUpdateList {
```

Comment:
Why is this needed? Haven't L210-L213 done it?

Reply:
> Why is this needed? Haven't L210-L213 done it?

Agreed, it is not needed.

Reply:
> is it still supposed to remove the endpointInstalledMap here given that you also do it in removeStaleEndpoints?
Here removes all endpoints related with a stale service. Function `removeStaleEndpoints` removes stale endpoints of a normal service.